core/math/aabb.h

```diff
@@ -118,6 +118,10 @@ class AABB {
 		return position + size;
 	}
 
+	_FORCE_INLINE_ Vector3 get_center() const {
+		return position + size * 0.5;
+	}
+
```

Comment:
```
position + (size * 0.5)
```
is better imo. In nearly all cases brackets are better for readability to show order of precedence (and to prevent future bugs).
See e.g.
https://softwareengineering.stackexchange.com/questions/201175/should-i-use-parentheses-in-logical-statements-even-where-not-necessary

Reply:
Done.
Did for C# too.